pkg/cmd/opts/install.go

```diff
@@ -227,31 +227,70 @@ func (o *CommonOptions) InstallKustomize() error {
 	if runtime.GOOS == "darwin" && !o.NoBrew {
 		return o.RunCommand("brew", "install", "kustomize")
 	}
+
 	binDir, err := util.JXBinLocation()
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "unable to find JXBinLocation")
 	}
-	fileName, flag, err := packages.ShouldInstallBinary("kustomize")
```

Comment:
BTW, _ShouldInstallBinary_ is not working for us. It does a check against PATH and returning _false_ if it finds anything anywhere on the PATH. Pretty pointless in our case. Maybe we can change this in a separate issue.

This part is problematic:

```
	pgmPath, err := exec.LookPath(fileName)
	if err == nil {
		log.Logger().Debugf("%s is already available on your PATH at %s", util.ColorInfo(fileName), util.ColorInfo(pgmPath))
		return
	}
```

Reply:
This was reason why the other test case I deleted was failing. ShouldInstallBinary was returning false even after adding binary at the place.

Reply:
Yeah, if you want we can look at this function and maybe fix it.